README.md

```diff
@@ -25,37 +21,62 @@ Build Wasm and native code:
 cargo build --release
 ```
 
-## Run
+Great! You have already compile the Centrifuge Chain!
 
-### Tests
+## Tests
 
-```bash
-cargo test -p centrifuge-runtime --release
-```
+There are two kinds of tests, one related to how the *Centrifuge Chain* works itself
+and another one to verify how it works in a more real environment as a parachain.
 
-### Start local Relay chain(alice and bob) and Parachain(alice)
+### Chain tests
 
-Prerequisites:
-- [docker](https://docs.docker.com/get-docker/)
-- [*jd*](https://stedolan.github.io/jq/)
+The following command will run unitary and integration tests:
 
-Start relay chain
 ```bash
-./scripts/init.sh start-relay-chain
-```
-
-Start  centrifuge-chain as parachain
-```bash
-./scripts/init.sh start-parachain
+cargo test -p centrifuge-runtime --release
```

Comment:
Also, this test is not actually testing anything:

```
running 1 test
test __construct_runtime_integrity_test::runtime_integrity_tests ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests centrifuge-runtime

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```


I think the best command would be

``` bash
cargo test --workspace --release --features test-benchmarks,try-runtime
```

Reply:
I have added `+nightly` to this command since there is some test code that uses nightly features that will not compile if you are using the stable channel.